.circleci/config.yml

```diff
@@ -112,6 +112,8 @@ jobs:
       # Pulls from Docker Hub may fail due to rate-limiting after Nov 1 2020.
       - sm-build/disable_docker_hub_access
       - sm-build/configure_gcr_docker_access
+      - setup_remote_docker
```

Comment:
Does this need to be performed before the configure gcr step and disable docker hub?  Not sure if docker configuration is clobbered.

Reply:
docker was not setup in the previous iteration. So, the order should not have created issues. But, I updated the order to maintain the correct sequence.